src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -1873,6 +1873,8 @@ public bool Translate(object sender, X509Certificate cert, X509Chain chain, SslP
                 lock (_locker)
                 {
                     _serverCertificateCustomValidationCallback = _serverCertificateCustomValidationCallback.Concat(new[] { value }).ToArray();
+                    GlobalHttpClientWithCompression.Clear();
```

Comment:
I do not see how does it help? Handler is attaching to `OnServerCertificateCustomValidationCallback` method, which is using the `_serverCertificateCustomValidationCallback`. Can you explain?

Reply:
If you already have an `HttpClient` for that thumbprint, you'll keep using this.
And will never re-create the `HttpHandler`, even if you create new `RequestExectuers`

Reply:
But why you need to recreate it?

Reply:
We are adding the callback only when we create the http handler, and only when creating the request executer.